subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -1,11 +1,17 @@
 # SPDX-License-Identifier: Apache-2.0
 
-if(CONFIG_BT_CTLR_ADVANCED_FEATURES)
+if (CONFIG_BT_CTLR_ADVANCED_FEATURES)
   message(WARNING "\nCONFIG_BT_CTLR_ADVANCED_FEATURES=y, Advanced Features' "
                   "default value change could change Zephyr Bluetooth "
                   "Controller's functional behavior.")
 endif()
 
+if (CONFIG_BT_CTLR_RNG_FAKE)
+  message(WARNING "\nCONFIG_BT_CTLR_RNG_FAKE is not a truly random generator. "
+                  "This capability is not secure and it is provided "
+                  "for testing purposes only. Use it carefully.")
```

Comment:
Suggest to also add `CONFIG_BT_CTLR_RNG_FAKE` to the `hardened.csv` file

Reply:
How come `EXPERIMENTAL` is not part of `hardened.csv`?

Reply:
No idea :)